binary/script/build_and_pr.sh

```diff
@@ -42,8 +41,11 @@ function check_github_token() {
 # check for changes to iPXE files
 function changes_detected() {
     local file="${1:-sha512sum.txt}"
+    local test="${file}.test" # .gitignore contains "*.test"
 
-    if sha512sum -c "${file}"; then
+    create_checksums "${test}"
+
+    if git diff --exit-code "${file}" "${test}" ; then
```

Comment:
`git diff` will just show history for these two files. probably need to switch to a regular `diff`.

Reply:
I did my experiment wrong. Let me fix this.